js/manager.js

```diff
@@ -150,11 +148,18 @@ export const DFPManager = Object.assign(new EventEmitter(), {
 
   registerSlot({ dfpNetworkId, adUnit, sizes, renderOutOfThePage, sizeMapping,
                  targetingArguments, slotId, slotShouldRefresh }) {
-    if (!registeredSlots.hasOwnProperty(slotId)) {
-      registeredSlots[slotId] = { slotId, sizes, renderOutOfThePage,
-                                  dfpNetworkId, adUnit, targetingArguments,
-                                  sizeMapping, slotShouldRefresh, loading: false,
-                                };
+    if (!registeredSlots[slotId]) {
```

Comment:
here what we really want is to check is property existence & precedence (regardless the value we want to be sure that prop isn't specified/defined at all) and ``` hasOwnProperty()``` seems like the best way to do it, Can u please revert to the previous approach?